kliff/transforms/configuration_transforms/graphs/helper.hpp

```diff
@@ -0,0 +1,116 @@
+// Author: Mingjian Wen (wenxx151@gmail.com)
```

Comment:
No need of symlinking, I had another solution in case this becomes untenable. I can just use setup.py to properly link to original files. Let me know if that works. Updated the setup.py

Reply:
That is even better!